### DISCUSSION TITLE
Restructure Well Bores to make it Object Oriented

### DISCUSSION THREAD

**Discussion by malcolm-dsider** [NatLabRockies/GEOPHIRES-X#77] [Ideas]
+Could we restructure this code to split it out into simpler and more logical objects, parents, children, etc.? Right now, it is pretty opaque. @kfbeckers

**Comment by softwareengineerprogrammer** [comment]
Treating this as addressed by https://github.com/NREL/GEOPHIRES-X/pull/3
